spec/xcpretty/parser_spec.rb

```diff
@@ -494,5 +494,17 @@ def given_kiwi_tests_are_done
       end
     end
 
+    context "informative" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.